bufflog.ts

```diff
@@ -5,7 +5,16 @@ export class BuffLog {
 
     constructor() {
         this.pinoLogger = require('pino')({
-            'level': this.defaultLevel,
+            level: this.defaultLevel,
+
+            // probably we want to call it `msg`. if so, let's change the PHP library instead
```

Comment:
@erickhun, did you land anywhere on this comment (re: message vs. msg)? I know msg is the default for pino and it's shorter. It also works nicely w/ [pino-pretty](https://github.com/pinojs/pino-pretty) right out of the box.

Reply:
@djfarrelly yeah, and I like it shorter too. opened an issue there   https://github.com/bufferapp/php-bufflog/issues/19

I'll see what I can do about it this week